pages/canvas/canvas_page.rb

```diff
@@ -531,17 +531,20 @@ def add_suite_c_tool(course, tool)
       end
     end
 
-    # Clicks the navigation link for a tool and returns the tool's URL
+    # Clicks the navigation link for a tool and returns the tool's URL. Optionally records an analytics event.
     # @param driver [Selenium::WebDriver]
     # @param tool [SuiteCTools]
+    # @param event [Event]
     # @return [String]
-    def click_tool_link(driver, tool)
+    def click_tool_link(driver, tool, event = nil)
       driver.switch_to.default_content
       hide_canvas_footer
       wait_for_update_and_click_js tool_nav_link(tool)
       wait_until(Utils.medium_wait) { title == "#{tool.name}" }
-      logger.info "#{tool.name} URL is #{current_url}"
-      current_url.delete '#'
+      logger.info "#{tool.name} URL is #{url = current_url}"
```

Comment:
I agree @johncrossman .  I plan to undo those eventually. I'm too obsessed with saving space.